addons/account/models/account_move.py

```diff
@@ -2389,7 +2336,7 @@ def _collect_tax_cash_basis_values(self):
     # -------------------------------------------------------------------------
 
     def _must_check_constrains_date_sequence(self):
-        return not self.posted_before and not self.quick_edit_mode
+        return not self.posted_before or not self.quick_edit_mode
```

Comment:
What's the rationale here? when do we want not to check the constraint? I feel like it's should be always checked :thinking:

Reply:
@qdp-odoo Fidu mode: as you can edit your sequence number before your date in fidu mode, you get excessive amounts of user errors triggered in the middle of your encoding, just because you're not encoding in order (which is a common use case for fiduciaries) https://www.odoo.com/web#id=2887007&menu_id=4720&cids=1&action=333&active_id=967&model=project.task&view_type=form

Reply:
This part is good for me.
But we can improve the behavior of the non blocking warning in order to keep it without it being annoying

Reply:
I'm ok with keeping the warning in some cases if @qdp-odoo / @tsb-odoo are ok with it :) See Wan comments below